src/Frame.h

```diff
@@ -98,7 +107,7 @@ namespace openshot
 	class Frame
 	{
 	private:
-		std::shared_ptr<QImage> image;
+		std::shared_ptr<QImage> image; ///< RGBA Format
```

Comment:
Remove this comment (and any similar ones), since our format is QImage::Format_RGBA8888_Premultiplied, it's a little misleading I think